api/src/main/java/io/opentelemetry/trace/samplers/DefaultDecision.java

```diff
@@ -26,7 +26,7 @@
  *
  * @since 0.1.0
  */
-public final class DefaultSamplingDecision implements Sampler.SamplingDecision {
+public final class DefaultDecision implements Decision {
```

Comment:
Is this kept public so it can be extended?  Other classes in this PR are package only visible.

Reply:
I have made it final for now, let' see if we want to extend it in SDK and change after.

Reply:
Consider to make it package protected for the moment.

Reply:
done